web/templates/settings/edit_custom_profile_field_form.hbs

```diff
@@ -20,34 +20,35 @@
             </div>
         </div>
     </div>
-    {{/if}}
-    {{#if is_external_account_field}}
-    <div class="input-group profile_field_external_accounts_edit">
-        <label for="external_acc_field_type">{{t "External account type" }}</label>
-        <select name="external_acc_field_type" class="modal_select" disabled>
-            {{#each ../realm_default_external_accounts}}
-                <option value='{{@key}}'>{{this.text}}</option>
-            {{/each}}
-            <option value="custom">{{t 'Custom' }}</option>
-        </select>
-    </div>
-    <div class="input-group custom_external_account_detail">
-        <label for="url_pattern">{{t "URL pattern" }}</label>
-        <input type="url" class="modal_url_input" name="url_pattern" autocomplete="off" maxlength="80" />
-    </div>
+    {{else if is_external_account_field}}
+    <span class="prop-element" id="id-custom-profile-field-field-data" data-setting-widget-type="field-data-setting">
```

Comment:
Wouldn't it be better to use a `div` here rather than `span` which is generally use to wrap inline elements.

Reply:
Fixed [here](https://github.com/zulip/zulip/compare/fb714ff6ffa7d12ade6dc359b4fdf7e77980fa77..8b36a3f8d4a74997d8760925d67f7f7fc646c0f9)